pkg/orderers/internal/common/pbftsegmentchkp.go

```diff
@@ -108,6 +108,11 @@ func (chkp *PbftSegmentChkp) NodeDone(nodeID types.NodeID, doneDigests [][]byte,
 	// Store Done message
 	chkp.doneReceived[nodeID] = struct{}{}
 	strKey := chkp.aggregateToString(doneDigests)
+
+	if _, ok := chkp.doneMsgIndex[strKey]; !ok {
+		chkp.doneMsgIndex[strKey] = []types.NodeID{nodeID}
+	}
+
```

Comment:
No need for this code. Appending to nil list works even without initialization.

Reply:
Thanks! done